packages/blockchain-wallet-v4-frontend/src/modals/GenerateReport/template.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+
+import { Image, Modal } from 'blockchain-info-components'
+
+import {
+  Description,
+  StyledButton,
+  StyledModalBody,
+  StyledModalFooter,
+  StyledModalHeader,
+  Title
+} from './model'
+
+const GeneratReport = ({ onClose }) => (
+  <Modal size='medium'>
+    <StyledModalHeader onClose={onClose} />
+    <StyledModalBody>
+      <Image name='subtract-check-circle' width='88px' height='88px' />
```

Comment:
Should the loader on the image be animated? I think it's just a static image right now?

Reply:
Yes, is a static image. Should I change anything? Feel free to suggest

Reply:
I will talk with UX about the icon. I think look great with some animation, but can be part of another pr